kubemarine/templates/plugins/sock-shop.yaml.j2

```diff
@@ -66,11 +66,11 @@ spec:
       nodeSelector:
         beta.kubernetes.io/os: linux
         {% if plugins['sock-shop']['nodeSelector'] is defined -%}
-          {{ plugins['sock-shop']['nodeSelector'] | toyaml | indent(width=8, indentfirst=False) -}}
```

Comment:
pls remove this file and it's referencies.

Reply:
delete files